src/main/java/propertyFiles/Bundle.properties

```diff
@@ -961,7 +960,6 @@ advanced.search.files.variableName=Variable Name
 advanced.search.files.variableName.tip=The name of the variable's column in the data frame.
 advanced.search.files.variableLabel=Variable Label
 advanced.search.files.variableLabel.tip=A short description of the variable.
-advanced.search.datasets.persistentId.tip=The persistent identifier for the Dataset.
 advanced.search.datasets.persistentId=Persistent Identifier
 advanced.search.datasets.persistentId.tip=The Dataset's unique persistent identifier, either a DOI or Handle
```

Comment:
Same idea - Given that we now allow PermaLinks and people can add new PID types, perhaps the shorter version is better? i.e. keep line 964 and remove this one?

Reply:
I think this would resolve https://github.com/IQSS/dataverse/issues/9740, too